src/cellfinder_core/__init__.py

```diff
@@ -1,4 +1,14 @@
-__version__ = "0.3.1"
+from importlib.metadata import PackageNotFoundError, version
+
+try:
+    __version__ = version("cellfinder-core")
+except PackageNotFoundError:
+    # Implying that the package we've just imported doesn't exist.
+    # Worrying, but throw an error here in this case
+    raise PackageNotFoundError(
+        "cellfinder-core name package is not present, but you appear to be importing it?"
+    )
```

Comment:
https://github.com/pypa/setuptools_scm#retrieving-package-version-at-runtime recommends just sticking a pass in here instead. I think an error would be fine, but would recommend
```suggestion
except PackageNotFoundError as e:
    raise PackageNotFoundError(
        "cellfinder-core package not installed"
    ) from e
```